File: dataset/output-bk/MISTRAL_codestral-latest/Task71_MISTRAL_codestral-latest.java
public class Main {
    public static void main(String[] args) {
        try {
            int num = Integer.parseInt(args[0]);
            System.out.println("The integer is: " + num);
        } catch (NumberFormatException e) {
            System.out.println("Invalid input. Please enter an integer.");
        }
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task71_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <cstdlib>

int main(int argc, char *argv[]) {
    char *end;
    long num = strtol(argv[1], &end, 10);

    if (*end != '\0' || num < INT_MIN || num > INT_MAX) {
        std::cout << "Invalid input. Please enter an integer.
";
    } else {
        std::cout << "The integer is: " << num << "
";
    }

    return 0;
}